js/components/App.react.js

```diff
@@ -5,7 +5,7 @@ var PluginListItem = require('./pluginListItem.react');
 
 var App = React.createClass({
 	getInitialState: function() {
-		return AppStore.getData();
+		return { data: AppStore.getData() };
```

Comment:
React won't let me return an array here, maybe you can help? or is this standard?